python/cuml/test/test_label_encoder.py

```diff
@@ -152,14 +152,18 @@ def test_empty_input(empty, ord_label):
 def test_masked_encode():
     int_values = [3, 1, 1, 2, 1, 1, 1, 1, 6, 5]
     cat_values = ['a', 'd', 'b', 'c', 'd', 'd', 'd', 'c', 'b', 'c']
-    df = cudf.DataFrame({"filter_col": int_values, "cat_col": cat_values})
+    df = cudf.DataFrame({"filter_col": int_values,
+                         "cat_col": cat_values})
 
     df_filter = df[df["filter_col"] == 1]
     df_filter["cat_col"] = LabelEncoder().fit_transform(df_filter["cat_col"])
 
-    filtered_int_values = [int_values[i] for i in range(len(int_values)) if int_values[i] == 1]
-    filtered_cat_values = [cat_values[i] for i in range(len(int_values)) if int_values[i] == 1]
-    df_test = cudf.DataFrame({"filter_col": filtered_int_values, "cat_col": filtered_cat_values})
+    filtered_int_values = [int_values[i] for i in range(len(int_values))
+                           if int_values[i] == 1]
+    filtered_cat_values = [cat_values[i] for i in range(len(int_values))
+                           if int_values[i] == 1]
+    df_test = cudf.DataFrame({"filter_col": filtered_int_values,
```

Comment:
In this case, you are creating a new dataframe, which will result in an index mismatch. So, when you try to assert they're equal you'd need to call `reset_index` (note that calling reset index always solved the problem because previously LabelEncoder was messing up the indices)

Reply:
@Nanthini10 Maybe this is better than reset_index? It emphasizes that values should match. Shall I update it that way?
`assert(df_filter["cat_col"].values == df_test["cat_col"].values).all()`

Reply:
Yeah, that's a better way to compare it. It'll ensure that NaN values aren't generated.

Reply:
I have updated it accordingly. Thanks @Nanthini10